source/OptimaJet.DWKit.StarterApplication/Models/Notification.cs

```diff
@@ -29,19 +26,10 @@ public class Notification : Identifiable, ITrackDate
         [Attr("date-updated")]
         public DateTime? DateUpdated { get; set; }
 
-        [Attr("message-substitutions"), NotMapped]
-        public object MessageSubstitutions { get; set; }
-
-        private const string EMPTY_JSON = "{}";
-        public string MessageSubstitutionsJson
-        {
-            get => (MessageSubstitutions == null)
-                ? EMPTY_JSON
-                : JsonConvert.SerializeObject(MessageSubstitutions);
-            set => MessageSubstitutions = string.IsNullOrWhiteSpace(value)
-                ? new Dictionary<string, object>()
-                : JObject.Parse(value).ToObject<Dictionary<string, object>>();
-        }
+        [Attr("message")]
+        public String Message { get; set; }
 
+        [Attr("send-email")]
+        public bool SendEmail { get; set; } = true;
```

Comment:
If you want the default to be true, then you also need a default in `AppDbContent.OnModelCreating`.  Otherwise, other mechanisms that could create a Notification (JsonApi or non-EF database interface) will not have the correct default.